File: app/src/main/java/com/arbaelbarca/listmovieskatalog/database/room/FavoriteDatabase.java
package com.arbaelbarca.listmovieskatalog.database.room;

import androidx.room.Database;
import androidx.room.RoomDatabase;
import androidx.room.TypeConverters;

import com.arbaelbarca.listmovieskatalog.model.movies.ResultsItem;

@Database(entities = {ResultsItem.class}, version = 3, exportSchema = false)
public abstract class FavoriteDatabase extends RoomDatabase {
    public abstract FavoriteDao favoriteDao();
}

File: app/src/main/java/com/robytakwa/listpicturekatalog/database/room/FavoriteDatabase.kt
package com.robytakwa.listpicturekatalog.database.room

import androidx.room.Database
import androidx.room.RoomDatabase
import com.robytakwa.listpicturekatalog.model.movies.ResultsItem

@Database(entities = [ResultsItem::class], version = 4, exportSchema = false)
abstract class FavoriteDatabase : RoomDatabase() {
    abstract fun favoriteDao(): FavoriteDao
}
